README.md

```diff
@@ -31,45 +22,50 @@ Technical documentation specific to the Sinopia BIBFRAME Editor may also be foun
 2.  Install [npm](https://www.npmjs.com/)
 3.  Run `npm init`, and follow the instructions that appear.
 4.  Get latest npm: `npm install -g npm@latest`
-5.  Run `npm install grunt-cli` to install grunt-cli.
-6.  Run `npm install`. This installs everything needed for the build to run successfully.
-7.  Run `npm run build` to build the code.  (TEMP: Run `npm run grunt` or `grunt` to build the BFE code.)
+5.  Run `npm run build` to build the code.
+
+### Installing BFE code
+1.  Run `npm install grunt-cli` to install grunt-cli.
+2.  Run `npm run grunt` or `grunt` to build the BFE code.
+3.  Run `npm install`. This installs everything needed for the build to run successfully.
+
 
 ## Running the code
 
 `npm start`
 
 Follow installation instructions, then run `npm start` or `node server.js` to start the web server using Express.
-This will start up the code at http://localhost:8000.
-
-TEMP: The Sinopia Editor code is currently available via http://localhost:8000/bfe-index.html.
+This will start up the code at [http://localhost:8000](http://localhost:8000).
 
-TEMP:  Note that `index.html` is meant to show the Sinopia home page via React.
+The Sinopia Editor code is currently available via [stage.sinopia.io][staging]
 
 ## Developers
 
 - See `package.json` for npm package dependencies.
-- The web server used is `express` web framework for node.js
+- The web server used is the `express` web framework for node.js
+- React components are located in `src/components/` directory
 
 ### Run the server with webpack-dev-webserver
 
 `npm run dev-start`
 
-This runs the webpack-dev-server, allowing immediate loading of live code changes without having to restart the server.
+Runs the webpack-dev-server, allowing immediate loading of live code changes
+without having to restart the server. The webpack-dev-server is available on at
+[http://localhost:8080](http://localhost:8080).
```

Comment:
same deal with hard returns -- are you sticking with an 80 char line or something like that?